### PR TITLE
chore: update EXPORTER_IMG var for Kepler

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR update the `make build-manifest` command to make use of Exporter image built using Prow CI instead of pulling from `quay`

### CODE REVIEW

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

**Comment by vprashar2929** [comment]
/pj-rehearse pull-ci-openshift-power-monitoring-kepler-main-kepler-e2e

File: ci-operator/config/openshift-power-monitoring/kepler/openshift-power-monitoring-kepler-main.yaml [hunk 1]
```diff
@@ -41,8 +41,11 @@ tests:
       cli: latest
       commands: |
         GOFLAGS="" make tools
-        make build-manifest OPTS="OPENSHIFT_DEPLOY"
+         GOFLAGS="" make build-manifest OPTS="OPENSHIFT_DEPLOY"
         oc apply -f _output/generated-manifest/deployment.yaml
+      dependencies:
+      - env: EXPORTER_IMG
+        name: kepler
```
Comment (sthaha) [inline ci-operator/config/openshift-power-monitoring/kepler/openshift-power-monitoring-kepler-main.yaml:48]: How is this being used? ... may be in a another PR, we should use a more precise name than exporter :)

File: ci-operator/config/openshift-power-monitoring/kepler/openshift-power-monitoring-kepler-main.yaml [hunk 2]
```diff
@@ -41,8 +41,11 @@ tests:
       cli: latest
       commands: |
         GOFLAGS="" make tools
-        make build-manifest OPTS="OPENSHIFT_DEPLOY"
+         GOFLAGS="" make build-manifest OPTS="OPENSHIFT_DEPLOY"
```
Comment (sthaha) [inline ci-operator/config/openshift-power-monitoring/kepler/openshift-power-monitoring-kepler-main.yaml:44]:
```suggestion
        GOFLAGS="" make build-manifest OPTS="OPENSHIFT_DEPLOY"
```

**Review by sthaha — COMMENTED** [review]
/lgtm
